js-miniapp-bridge/src/types/error-types/index.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Enum for supported SDK error types
+ */
+export enum MiniAppErrorType {
+  AudienceNotSupportedError = 'AudienceNotSupportedError',
+  ScopesNotSupportedError = 'ScopesNotSupportedError',
+  AuthorizationFailureError = 'AuthorizationFailureError',
+  Other = 'Other',
```

Comment:
Is `Other` actually being used anywhere? It doesn't look like it's being used anymore, so maybe it can be removed.

Reply:
it's true it's not, I remove it